### ISSUE TITLE
npm ERR! Error: EACCES, mkdir '/Users/root'

### ISSUE THREAD

**Issue by sporkd** [boxen/puppet-nodejs#54]
Node versions seem to install fine, but the modules keep failing with the following error:

```
Error: Execution of '/opt/nodes/0.10/bin/npm install 'gulp'@'~> 3.9.0' --global' returned 3: npm ERR! Darwin 14.5.0
npm ERR! argv "node" "/opt/nodes/0.10/bin/npm" "install" "gulp@~> 3.9.0" "--global"
npm ERR! node v0.10.36
npm ERR! npm  v2.14.2
npm ERR! path /Users/root
npm ERR! code EACCES
npm ERR! errno 3

npm ERR! Error: EACCES, mkdir '/Users/root'
npm ERR!  { [Error: EACCES, mkdir '/Users/root'] errno: 3, code: 'EACCES', path: '/Users/root' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.
```

However, if I copy paste into terminal just the actual command boxen says it's running, then it works fine with no permissions errors.

```
% /opt/nodes/0.10/bin/npm install 'gulp'@'~> 3.9.0' --global

... WORKS!!!

gulp@3.9.0 /opt/boxen/nodenv/versions/0.10/lib/node_modules/gulp
```

BTW, My module definition looks like this:

```
  npm_module { 'gulp for all nodes':
    module        => 'gulp',
    node_version  => '*',
    version       => '~> 3.9.0'
  }
```

**Comment by sporkd** [comment]
Looking through closed PRs and ran across this https://github.com/boxen/puppet-nodejs/pull/52

Probably just hasn't been released as a version yet.

**Comment by blackjid** [comment]
Yes @sporkd sorry, I made a mistake when I released `5.0.2` Now the fix should be in `5.0.3`
Please fill free to reopen if something is wrong..

**Comment by sporkd** [comment]
No worries. Thanks for the quick update @blackjid. Looks like it's working now.
